### PR TITLE
Refactor include file disk dumping

### PR DESCRIPTION
**Pull request by berland**
Improve tests on the way

### CODE REVIEW

File: ecl2df/common.py [hunk 1]
```diff
@@ -84,7 +85,7 @@ def write_dframe_stdout_file(
         caller_logger (logging): Used if not stdout
         logstr (str): Logged if not stdout.
     """
-    if output == "-":
+    if output == __MAGIC_STDOUT__:
```
Comment (jondequinor) [inline ecl2df/common.py:88]: I think `None` would work just as well here? `-` usually is `stdin` in bash, might cause some confusion?

File: ecl2df/common.py [hunk 2]
```diff
@@ -66,6 +64,9 @@
     )
 
 
+# This is a magic filename that means read/write from/to stdout:
+__MAGIC_STDOUT__ = "-"
```
Comment (jondequinor) [inline ecl2df/common.py:68]: Why the double dunder? Does this mean I cannot do this?

```python
from ecl2df.common import __MAGIC_STDOUT__

…
write_dframe_stdout_file(my_frame, __MAGIC_STDOUT__)
```

If that does not work, this is unfortunate.

File: ecl2df/common.py (hunk 1 above)
Comment (jondequinor) [reply ecl2df/common.py:88]: Also, isn't `-` a perfectly valid file name?

**Review by jondequinor — CHANGES_REQUESTED** [review]
Strongly recommend changing the `write_dframe_stdout_file` API.

File: ecl2df/common.py (hunk 1 above)
Comment (berland) [reply ecl2df/common.py:88]: It works, but it creates a lot of problems. Many bash tools cannot create the file, `touch -` will not work. `echo foo > -` works, but `cat -` will not work afterwards. Not even `cat -- -`. You can also try `mkdir -` but it is hard to `cd -` afterwards.

File: ecl2df/common.py (hunk 2 above)
Comment (berland) [reply ecl2df/common.py:68]: yes, that ddunder is probably misplaced.

**Pull request merged**